app/templates/register.hbs

```diff
@@ -85,42 +86,45 @@
                         <hr>
                         <div class="col-lg-6">
                             <label for="city" class="form-control-label">Start</label>
-                            {{date-picker date=model.startDate class="form-control form-spacing" valueFormat='date' id='startDate'}}
-                            {{#if this.isInvalidDates}}
-                                <br>
-                                <div class="warning">Start date must be before end date</div>
-                            {{/if}}
+                            {{date-picker date=model.startDate class="form-control" valueFormat='date' id='startDate'}}
                         </div>
-                        <div class="col-md-6">
+                        <div class="col-lg-6">
```

Comment:
Yeah, the problem was that on larger screens, the start and end date text fields were different sizes. Now when they expand and compress, they remain they stay consistent.